solutions/agents/variables.tf

```diff
@@ -41,82 +41,82 @@ variable "cluster_config_endpoint_type" {
 
 variable "log_analysis_enabled" {
   type        = bool
-  description = "Deploy IBM Cloud Logging agent if set as true."
+  description = "Whether to deploy the IBM Cloud logging agent."
   default     = true
 }
 
 
 variable "log_analysis_agent_tags" {
   type        = list(string)
-  description = "List of tags to associate to all log records that the agent collects so that you can identify the agent's data quicker in the logging UI. NOTE: Use the 'log_analysis_add_cluster_name' variable to add the cluster name as a tag."
+  description = "The list of tags to associate with all log records collected by the agent so that you can quickly identify the agent’s data in the logging UI. To add the cluster name as a tag, use the `log_analysis_add_cluster_name` variable."
   default     = []
   nullable    = false
 }
 
 variable "log_analysis_add_cluster_name" {
   type        = bool
-  description = "If true, configure the log analysis agent to attach a tag containing the cluster name to all log messages."
+  description = "Whether to attach the cluster name to log messages. Set to `true` to configure the IBM Log Analysis agent to tag all log messages with the name."
   default     = true
 }
 
 variable "log_analysis_ingestion_key" {
   type        = string
-  description = "Ingestion key for the IBM Cloud Logging agent to communicate with the instance."
+  description = "The ingestion key that is used by the IBM Cloud logging agent to communicate with the instance."
   sensitive   = true
   default     = null
 }
 
 variable "log_analysis_secret_name" {
   type        = string
-  description = "The name of the secret which will store the ingestion key. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the secret that stores the ingestion key. If a prefix input variable is specified, the secret name is prefixed to the value in the `<prefix>-value` format."
```

Comment:
We missed this one: it isn't using the `<prefix>-name` update.

Reply:
Thanks @SirSpidey I've updated  it to use `<prefix>-name`.